sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -542,6 +544,60 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
         }
         checkPushedInfo(df8, expectedPlanFragment8)
         checkAnswer(df8, Seq(Row(2, "david", 10000, 1300, true)))
+
+        val df9 = spark.table("h2.test.employee")
+          .filter(logarithm(2, $"dept") > 2)
+          .filter(log10($"dept") > 0.5)
+          .filter(round($"dept") > 1)
+          .filter(degrees($"dept") > 100)
+          .filter(radians($"dept") > 0.1)
+          .filter(signum($"dept") === 1)
+        checkFiltersRemoved(df9, ansiMode)
```

Comment:
These are not ansi functions, so the push down shouldn't be affected by `ansiMode`?

Reply:
The reason is some function uses catalyst expression that affected by `ansiMode`.

Reply:
I think you have `Cast`, which is affected by `ansiMode`.  I feel it is cleaner to only test the newly added non-ansi functions, without mixing with the ansi functions.

Reply:
`Logarithm` uses / also affected by `ansiMode`.
```
case class Logarithm(left: Expression, right: Expression)
  extends BinaryMathExpression((c1, c2) => StrictMath.log(c2) / StrictMath.log(c1), "LOG")
```